src/legacy/core_plugins/telemetry/public/hacks/fetch_telemetry.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import uiChrome from 'ui/chrome';
+import moment from 'moment';
+
+/**
+ * Fetch Telemetry data by calling the Kibana API.
+ *
+ * @param {Object} $http The HTTP handler
+ * @param {String} basePath The base URI
+ * @param {Function} _moment moment.js, but injectable for tests
+ * @return {Promise} An array of cluster Telemetry objects.
+ */
+export function fetchTelemetry($http, { basePath = uiChrome.getBasePath(), _moment = moment, unencrypted = false } = { }) {
+  return $http.post(`${basePath}/api/telemetry/v2/clusters/_stats`, {
+    unencrypted,
+    timeRange: {
+      min: _moment().subtract(20, 'minutes').toISOString(),
+      max: _moment().toISOString()
```

Comment:
Should this time range maybe honor timezone offset set in Management > Advanced Settings? Same for https://github.com/elastic/kibana/pull/45769/files#diff-cd77b28838727bb5489668ca02f97e0eR65-R68

Might be a separate issue/PR though

Reply:
This is used to query ES monitoring data captured in the last 20 minutes.

I am not sure what are the implications of changing that setting and what data we get from monitoring. I'll try changing the setting and see if that affects us.

If anything needs to be changed here, maybe changing it to `now` and `now-20m` when querying es instead of absolute dates would be best here.

Good catch!

Reply:
I'll open an issue about this for a future PR after further investigating. The PR is already HUGE so i dont want add more scope.

Reply:
I'm not sure I follow here. The timestamps used for searching against ES are all UTC-based. The only time we need to care about the `dateFormat:tz` config is when we are showing the time to the user, which I don't think we're doing here?

Reply:
@chrisronline You're absolutely right! I forgot that `toISOString()` converts to UTC (and if the timestamp is not displayed anywhere no need to convert)

@Bamieh Scratch my concern here, and sorry for the derp moment (timezones are hard). I feel sorry for future generations that will also need to support the Mars timezones :trollface: